src/types/method.ts

```diff
@@ -0,0 +1,29 @@
+export type Method = {
+  name: string;
+  namespace: MethodNamespace;
+  description: string;
+  parameters?: MethodVariable[];
+  return: MethodVariable;
+  example?: MethodExample;
```

Comment:
Let's included references here too
```suggestion
  example?: MethodExample;
  references: string[];
```

Reply:
Added! https://github.com/mds1/evm-diff/pull/52/commits/fa9603508cd77737596a918cfbaa61abdadd9b58